packages/fcl/src/current-user/normalize/local-view.js

```diff
@@ -0,0 +1,32 @@
+import {LOCAL_VIEW_PRAGMA} from "./__vsn"
+
+// {
+//    "f_type": "Service",
+//    "f_vsn": "1.0.0",
+//    type: "local-view",
+//    method: "VIEW/IFRAME",
+//    endpoint: "https://woot.org/authz/local",
+//    data: {},
+//    params: {},
+// }
+export function normalizeLocalView(resp) {
+  if (resp == null) return null
+  if (resp.method == null) {
+    resp = {...resp, type: "local-view", method: "VIEW/IFRAME"}
```

Comment:
Maybe....
`resp = {type: "local-view", method: "VIEW/IFRAME", ...rep}`
Otherwise they are all going to be iframes...

Reply:
yah, that would default to iframe for service without `method`